graph-builder/src/graph.rs

```diff
@@ -164,9 +164,9 @@ impl HasRegistry for State {
 }
 
 #[allow(clippy::useless_let_if_seq)]
-pub async fn run<'a>(settings: &'a config::AppSettings, state: &State) -> ! {
```

Comment:
Not sure why lifetime is changed here

Reply:
This was part of the compiler warning fix. I noticed it while working on this file.